core/src/main/java/io/questdb/griffin/engine/functions/conditional/NullIfIFunctionFactory.java

```diff
@@ -30,16 +30,16 @@
 import io.questdb.griffin.FunctionFactory;
 import io.questdb.griffin.PlanSink;
 import io.questdb.griffin.SqlExecutionContext;
+import io.questdb.griffin.engine.functions.BinaryFunction;
 import io.questdb.griffin.engine.functions.IntFunction;
-import io.questdb.griffin.engine.functions.UnaryFunction;
 import io.questdb.std.IntList;
 import io.questdb.std.Numbers;
 import io.questdb.std.ObjList;
 
```

Comment:
Could you : 
- rename it to NullIfIntFunctionFactory  
- add test with two non-constant int arguments ?

Reply:
Sure. Just pushed the requested changes. Thanks!